client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java

```diff
@@ -434,6 +435,32 @@ void shouldBeAbleToEnableMutiny() throws URISyntaxException, FileNotFoundExcepti
 
     }
 
+    @Test
+    void shouldBeAbleToApplyMutinyOnSpecificEndpoints() throws URISyntaxException, FileNotFoundException {
```

Comment:
Testing a bad configuration from the user, I got this: `expected: <io.smallrye.mutiny.Uni<String>> but was: <String>`

Reply:
it must be the old default behavior (`io.smallrye.mutiny.Uni<String>`), no?

Reply:
what do u expect if the configuration is wrong ? throw an exception ? if yes which one ?

Reply:
I confirm that when the configuration is wrong we fallback to the old behavior : 
```
 {#else}
                        {#if return-response}
    public jakarta.ws.rs.core.Response {op.nickname}(
                        {#else}
                            {#if op.returnType == "void"}
    public jakarta.ws.rs.core.Response {op.nickname}(
                            {#else}
    public {#if op.returnType}{op.returnType}{#else}jakarta.ws.rs.core.Response{/if} {op.nickname}(
                            {/if}
                        {/if}
                    {/if}
```

I can add the case when `#if mutiny-operation-ids.get(op.operationIdOriginal)` return : 
```

 {#if return-response}
    public io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response> {op.nickname}(
                    {#else}
                        {#if op.returnType == "void"}
    public io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response> {op.nickname}(
                        {#else}
    public io.smallrye.mutiny.Uni<{#if op.returnType}{op.returnType}{#else}jakarta.ws.rs.core.Response{/if}> {op.nickname}(
                        {/if}
                    {/if}
```

what do u thing ?

Reply:
Executing this test without your changes (main branch), we have the return type as `Uni<String>`:

<details>
<summary>shouldBeAbleToApplyMutinyOnSpecificEndpoints ✅ </summary>

```java
@Test
    void shouldBeAbleToApplyMutinyOnSpecificEndpoints() throws URISyntaxException, FileNotFoundException {
        List<File> generatedFiles = createGeneratorWrapper("simple-openapi.json")
                .withMutiny(true)
                .generate("org.mutiny.enabled");

        Optional<File> file = generatedFiles.stream()
                .filter(f -> f.getName().endsWith("DefaultApi.java"))
                .findAny();
        assertThat(file).isNotEmpty();
        CompilationUnit compilationUnit = StaticJavaParser.parse(file.orElseThrow());
        List<MethodDeclaration> methodDeclarations = compilationUnit.findAll(MethodDeclaration.class);
        assertThat(methodDeclarations).isNotEmpty();

        Optional<MethodDeclaration> helloMethodDeclaration = getMethodDeclarationByIdentifier(methodDeclarations,
                "helloMethod");
        Optional<MethodDeclaration> byeMethodGetDeclaration = getMethodDeclarationByIdentifier(methodDeclarations,
                "byeMethodGet");

        helloMethodDeclaration.ifPresent(
                methodDeclaration -> assertEquals("io.smallrye.mutiny.Uni<String>", methodDeclaration.getType().toString()));
        byeMethodGetDeclaration.ifPresent(
                methodDeclaration -> assertEquals("io.smallrye.mutiny.Uni<String>", methodDeclaration.getType().toString()));
    }

    private static Optional<MethodDeclaration> getMethodDeclarationByIdentifier(List<MethodDeclaration> methodDeclarations,
                                                                                String methodName) {
        return methodDeclarations.stream().filter(md -> md.getName().getIdentifier().equals(methodName)).findAny();
    }
```
</details>

Executing this test in your branch, I got it `<String>` not `io.smallrye.mutiny.Multi<String>`:

<details>
<summary>inYourBranch ✅</summary>

```java
@Test
    void shouldBeAbleToApplyMutinyOnSpecificEndpoints() throws URISyntaxException, FileNotFoundException {
        List<File> generatedFiles = createGeneratorWrapper("simple-openapi.json")
                .withMutiny(true)
                .withMutinyReturnTypes(Map.of("helloMethod", "BadConfig", "Bye method_get", "Multi"))
                .generate("org.mutiny.enabled");

        Optional<File> file = generatedFiles.stream()
                .filter(f -> f.getName().endsWith("DefaultApi.java"))
                .findAny();
        assertThat(file).isNotEmpty();
        CompilationUnit compilationUnit = StaticJavaParser.parse(file.orElseThrow());
        List<MethodDeclaration> methodDeclarations = compilationUnit.findAll(MethodDeclaration.class);
        assertThat(methodDeclarations).isNotEmpty();

        Optional<MethodDeclaration> helloMethodDeclaration = getMethodDeclarationByIdentifier(methodDeclarations,
                "helloMethod");
        Optional<MethodDeclaration> byeMethodGetDeclaration = getMethodDeclarationByIdentifier(methodDeclarations,
                "byeMethodGet");

        helloMethodDeclaration.ifPresent(
                methodDeclaration -> assertEquals("String", methodDeclaration.getType().toString()));
// @oben01  I think must be io.smalrye.mutiny.Multi<String> because we are using a wrong configuration (`BadConfig`) 
        byeMethodGetDeclaration.ifPresent(
                methodDeclaration -> assertEquals("io.smallrye.mutiny.Multi<String>", methodDeclaration.getType().toString()));
    }
```
</details>

Reply:
if i understand correctly when there is a bad configuration we should fallback to io.smallrye.mutiny.Uni<String> in your example ? I am missing something ?

Reply:
@mcruzdev i'll do the changes !

Reply:
In other words, this test must be ✅ 

```java
@Test
    void shouldBeAbleToApplyMutinyOnSpecificEndpointsWhenUserDefineWrongConfiguration() throws URISyntaxException, FileNotFoundException {
        List<File> generatedFiles = createGeneratorWrapper("simple-openapi.json")
                .withMutiny(true)
                .withMutinyReturnTypes(Map.of("helloMethod", "Uni", "Bye method_get", "BadConfig"))
                .generate("org.mutiny.enabled");

        Optional<File> file = generatedFiles.stream()
                .filter(f -> f.getName().endsWith("DefaultApi.java"))
                .findAny();
        assertThat(file).isNotEmpty();
        CompilationUnit compilationUnit = StaticJavaParser.parse(file.orElseThrow());
        List<MethodDeclaration> methodDeclarations = compilationUnit.findAll(MethodDeclaration.class);
        assertThat(methodDeclarations).isNotEmpty();

        Optional<MethodDeclaration> helloMethodDeclaration = getMethodDeclarationByIdentifier(methodDeclarations,
                "helloMethod");
        Optional<MethodDeclaration> byeMethodGetDeclaration = getMethodDeclarationByIdentifier(methodDeclarations,
                "byeMethodGet");

        helloMethodDeclaration.ifPresent(
                methodDeclaration -> assertEquals("io.smallrye.mutiny.Uni<String>", methodDeclaration.getType().toString()));
        byeMethodGetDeclaration.ifPresent(
                methodDeclaration -> assertEquals("io.smallrye.mutiny.Uni<String>", methodDeclaration.getType().toString()));
    }
```

Sorry for type a lot 😞

Reply:
Could you share your thoughts on the following behavior?

- When Mutiny=true, we apply Uni to all operationIds.

- When mutiny.operation-ids.addPet=Uni, we apply Uni only to the addPet operation, keeping the default behavior for other operationIds.

- When mutiny.operation-ids.addPet=WRONG, we still apply Uni to addPet, while other operationIds follow the normal behavior.

Does this approach align with your expectations, or would you suggest any adjustments?